viewmodelevents/README.md

```diff
@@ -0,0 +1,194 @@
+# ViewModelEvents <GitHub path="RedMadRobot/gears-android/tree/main/ktx/viewmodel-events-ktx"/>
```

Comment:
This tag is not needed anymore. It was previously required for rendering in our knowledge base
```suggestion
# ViewModelEvents
```